test/versioned/redis/redis-v4-legacy-mode.tap.js

```diff
@@ -30,7 +30,8 @@ test('Redis instrumentation', function (t) {
     const redis = require('redis')
     client = redis.createClient({
       legacyMode: true,
-      socket: { port: params.redis_port, host: params.redis_host }
+      port: params.redis_port,
+      host: params.redis_host
```

Comment:
For future readers: the alternate form is tested in another suite: https://github.com/newrelic/node-newrelic/blob/b4b6a6b2eca8bd47d17f8b265344b4596c8226b3/test/versioned/redis/redis-v4.tap.js#L30-L31